projects/rawkode-academy/image-service/src/pages/image.ts

```diff
@@ -87,6 +87,11 @@ const getPayloadFromSearchParams = (searchParams: URLSearchParams): Payload => {
 	return DEFAULT_PAYLOAD;
 };
 
+// cache for one day
+// When changing to higher caching times, consider adding "immutable, no-transform"
+
+const CACHE_CONTROL = "public, max-age=86400";
```

Comment:
Something to think about for later, but can we invalidate the cache if the templates change?

Perhaps we can use some sort of etag

Reply:
Yeah, `etag` on the hash of the content maybe or the modification date